src/components/visual-submit/component.tsx

```diff
@@ -59,8 +59,50 @@ function PluginVisualSubmit({ pluginUuid }: PluginVisualSubmitProps): React.Reac
 
   const {
     pushEntry: pushSubmitImage,
+    data: submitImageResponseData,
+    replaceEntry: updateSubmitImage,
   } = pluginApi.useDataChannel<SubmitImage>('submitImage', DataChannelTypes.ALL_ITEMS);
 
+  const submittedImages = submitImageResponseData?.data || [];
+
+  React.useEffect(() => {
+    if (updateSubmitImage && !currentUser?.presenter) {
+      const userSubmitCount: {[key: string]: number} = {};
+      submittedImages.filter((submittedImage) => {
+        if (
+          submittedImage.payloadJson.renderedInLAD
+        ) {
+          const currentUserSubmitCount = userSubmitCount[
+            submittedImage.payloadJson.submittedBy.userId]
+            ? userSubmitCount[submittedImage.payloadJson.submittedBy.userId] : 0;
+          userSubmitCount[
+            submittedImage.payloadJson.submittedBy.userId
+          ] = currentUserSubmitCount + 1;
+        }
+        return !submittedImage.payloadJson.renderedInLAD;
+      }).forEach(
+        (submittedImage) => {
+          const submitNumber = userSubmitCount[submittedImage.payloadJson.submittedBy.userId]
+            ? userSubmitCount[submittedImage.payloadJson.submittedBy.userId] : 0;
+          const submitColumnTitle = `Submit ${submitNumber + 1}`;
+          pluginApi.sendGenericDataForLearningAnalyticsDashboard({
+            columnTitle: submitColumnTitle,
+            value: `[Submit](${submittedImage.payloadJson.imageUrl})`,
```

Comment:
I suggest sending as image and setting the proper `alt` name.

```suggestion
            value: `![Submit ${submitNumber + 1}](${submittedImage.payloadJson.imageUrl})`,
```